rules.ini

```diff
@@ -11,6 +11,7 @@ Build[] = (?:^|/)COMMIT\.DAT$
 Build[] = (?:^|/)GAME\.CON$
 CFG = \.cfg$ ; idTech1
 DATA_WIN = (?:^|/)data\.win$ ; GameMaker
+DATA_DB_INDEX = (?:^|/)data/db/index$ ; Stride
```

Comment:
```suggestion
```
detecting the engine via its dll is enough, we don't need evidences here

Reply:
I'm ok to remove it. However, people may use a single file publishing method that would pack all the DLLs into the executable. In that case there is no DLLs to detect. But content files would still be separately included in the `data` folder.

Reply:
@Lifeismana waiting for a comment from you, should I nonetheless remove this?